docs/moderator-handbook.md

```diff
@@ -229,15 +229,15 @@ Here's how moderators deal with violations of our [Code of Conduct](https://code
    Evidence: _One or more links to the offending message(s)_
    ```
 
-3. **Creating a Private Discussion**
+3. **Creating a private discussion**
 
    There may be situations where you need to address a concern with a camper privately. This should not be done through DMs, which can lead to situations where you claim one thing and the camper claims another. Instead, use the bot's functionality to create a private discussion:
 
-   - Call the `!fCC private username` command, where `username` is the camper's chat username.
+   - Call the `!fcc private username` command, where `username` is the camper's chat username.
    - The bot will create a new channel, and add the mentioned camper and all moderators with the `Your Friendly Moderator` role. While all moderators are added to the channel for transparency, the moderator who calls this command should be the only one to interact with the camper unless they request assistance.
-   - When the conversation is complete, call the `!fCC close` command _in the private channel_ to have the bot close and delete that channel.
+   - When the conversation is complete, call the `!fcc close` command _in the private channel_ to have the bot close and delete that channel.
```

Comment:
Note: this needs to be updated
cc/ @naomi-lgbt

Reply:
```suggestion
   - When the conversation is complete, click the `❌ Close` button _on the first message in the private channel_ to have the bot close and delete that channel.
```